example.py

```diff
@@ -77,13 +77,11 @@ def ac_mon(client, device_id):
     its status such as its temperature and operation mode.
     """
 
-    device = client.get_device(device_id)
-    if device.type != wideq.DeviceType.AC:
+    ac = client.get_device_class(device_id)
+    if not isinstance(ac, 'ACDevice'):
```

Comment:
Shouldn't this be `isinstance(ac, wideq.ACDevice)` (instead of passing a string)?

Reply:
not so low-level suggestion! I tested it with 'mon' argument instead of 'ad-mon' and can't imagine why there is a need of different keyword for the same command. So, I removed ac-mon :) and made gen_mon or ac_mon monitoring methods, selection is made according to the type of device_class, hope you will agree with this ?

Reply:
Yeah, that seems good to me!